wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+
+#### Điểm số được đặt ở chế độ ổn định cuối cùng sẽ hiển thị ở lazer?
+
+Đúng. Sau khi chúng tôi hoàn tất việc cân bằng sự kết hợp giữa lazer và điểm số ổn định, cả hai sẽ hiển thị.
+
+#### Tất cả các mod sẽ được xếp hạng?
+
+Hiện tại, điểm số của tất cả các kết hợp mod xuất hiện trên bảng xếp hạng. Liệu điểm số có cho điểm hiệu suất với tất cả các mod hay không (và nếu có, liệu sẽ có phần thưởng hay hình phạt được áp dụng hay không) vẫn còn đang được thảo luận.
+
+#### Tôi không thích cơ chế chơi trò chơi mới. Tôi có thể khôi phục cơ chế chơi trò chơi cũ như ở phiên bản ổn định không?
+
+Vui lòng thử mod "cổ điển", mod này sẽ khôi phục phần lớn hành vi cũ mà bạn đã quen. Ngoài ra, hãy đảm bảo kiểm tra các cài đặt do mod cổ điển cung cấp, vì nó sẽ cho phép bạn tùy chỉnh thêm trải nghiệm của mình và cũng hiểu những thay đổi nào đang được áp dụng (vì tất cả chúng đều được liệt kê ở đó).
+
+### Skinning và giao diện người dùng
+
+#### Một cái gì đó đang hoạt động khác với ổn định và tôi không thích nó!
+
+Vui lòng chạy trình hướng dẫn thiết lập ở đầu cài đặt và xem qua các cài đặt trên màn hình `Hành vi`. Rất nhiều cài đặt phổ biến đã thay đổi giá trị mặc định được liệt kê ở đây. Ngoài ra còn có một nút duy nhất mà bạn có thể nhấn để áp dụng các hành vi cũ làm điểm khởi đầu cho hành trình lazer của mình.
+
+#### Giao diện cũ cuối cùng có hoạt động trong màn hình chọn bài hát và kết quả không?
+
+Chúng tôi sẽ cố gắng hết sức để khôi phục lại nhiều thứ nhất có thể mà không chặn chức năng mới. Điều này sẽ đến sau này.
+
+#### Tôi có thể sử dụng con trỏ skin của mình trong menu không?
+
+Chúng tôi có thể sẽ mang lại hỗ trợ cho điều này trong tương lai do nhu cầu phổ biến.
+
+### Hiệu suất
+
+#### Tại sao tôi không thể sử dụng với không giới hạn FPS?
+
+Trên một ngưỡng nhất định, không có lý do gì để chạy ở tốc độ khung hình cao hơn. Lazer sử dụng nhiều công nghệ mới khác nhau để đảm bảo có thể đạt được độ trễ thấp nhất mà không yêu cầu tốc độ khung hình cao. Điều này sẽ tiếp tục cải thiện trong tương lai vì chúng tôi vẫn còn một số cải tiến cần thực hiện.
+
+Các cuộc thăm dò của Lazer cho đầu vào ở 1000 Hz bất kể bộ giới hạn FPS, đó là lý do tại sao cài đặt bộ giới hạn tối đa cũng sẽ giới hạn ở 1000 FPS.
+
+Nếu bạn tò mò về cách điều này ảnh hưởng đến độ trễ đầu vào và kiểm tra, vui lòng chạy "trình xác nhận độ trễ" tích hợp ở cuối mục cài đặt.
+
+#### Nếu đầu vào chỉ được thăm dò ở 1000 Hz, vậy còn con chuột chơi game 8000 Hz của tôi thì sao?
+
+Hệ điều hành sẽ vẫn thăm dò ở tỷ lệ cao hơn, mặc dù lợi ích được chứng minh là không đáng kể. Việc thăm dò ý kiến ở tốc độ cao như vậy có thể gây ra các chi phí không lường trước được và chúng tôi khuyên bạn nên giới hạn các thiết bị ở tần số 1000 Hz để hệ thống ổn định.
```

Comment:
```suggestion
Hệ điều hành sẽ vẫn poll ở tỷ lệ cao hơn, mặc dù lợi ích được chứng minh là không đáng kể. Việc poll ở tốc độ cao như vậy có thể gây ra các overhead không lường trước được và chúng tôi khuyên bạn nên giới hạn các thiết bị ở tần suất 1000 Hz để hệ thống ổn định.
```
Where does the word `ý kiến` comes from?
And `rate` should be translated to `tần suất`. `Tần số` is used for designated frequencies in radio usage, etc...